### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by abompard**
This set of commits should fix the integrations tests and make them pass again.

### CODE REVIEW

**Comment by abompard** [comment]
For some reason the F35 and rawhide builds fail with this error when installing packages:
```
Errors during downloading metadata for repository 'rawhide':
- Curl error (6): Couldn't resolve host name for https://mirrors.fedoraproject.org/metalink?repo=rawhide&arch=x86_64 [getaddrinfo() thread failed to start]
Error: Failed to download metadata for repo 'rawhide': Cannot prepare internal mirrorlist: Curl error (6): Couldn't resolve host name for https://mirrors.fedoraproject.org/metalink?repo=rawhide&arch=x86_64 [getaddrinfo() thread failed to start]
```

The CI VM can't resolve mirrors.fp.o? That's weird, any idea what's going on?

**Comment by abompard** [comment]
One of the commits adjusts for a possible change in Greenwave's return structure, that's the one I would most like to have a review one. I guessed the error came from a change in Greenwave's return structure but I may have been wrong there, as I haven't followed it closely.

File: devel/ci/integration/tests/test_bodhi_cli.py [hunk 1]
```diff
@@ -477,13 +478,8 @@ def test_updates_download(bodhi_container, db_container):
...
     # something we can track.
     koji_mock = "#!/bin/sh\necho TESTING CALL $0 $@\n"
-    # we mock-replace the executable in two places
-    # It's because of https://pagure.io/koji/issue/912
-    # Koji is installed from RPM isn't found because it doesn't provide egg info.
-    # So Bodhi will install Koji from Pypi and the Koji-Pypi executable is used.
     with replace_file(bodhi_container, "/usr/bin/koji", koji_mock):
-        with replace_file(bodhi_container, "/usr/local/bin/koji", koji_mock):
-            result = _run_cli(bodhi_container, cmd)
+        result = _run_cli(bodhi_container, cmd)
```
Comment (mattiaverga) [inline devel/ci/integration/tests/test_bodhi_cli.py:482]: So, can we also remove the `pip install koji` line from dockerfiles?

File: devel/ci/integration/tests/test_bodhi_tasks.py [hunk 2]
```diff
@@ -138,18 +140,17 @@ def find_bug():
...
         "--bugs",
         bug_id,
+        "--autokarma",
         update_alias,
     ]
-    celery_tasks = bodhi_container.execute(["ls", "/srv/celery-results"])
-    print(list(celery_tasks))
     try:
-        bodhi_container.execute(cmd)
+        result = bodhi_container.execute(cmd)
```
Comment (mattiaverga) [inline devel/ci/integration/tests/test_bodhi_tasks.py]: I'm just pedantic here: is the result variable used anywhere?

**Review by mattiaverga — COMMENTED** [review]
Thank you for working on this! Yes, there were some changes in greenwave stuff, but just like the EOL field for releases these changes weren't reflected in the integration tests, because no one ever looked at their unreliable results.

I've just left a couple of pedantic comments, but overall everything looks fine.

File: devel/ci/integration/tests/test_bodhi_cli.py (hunk 1 above)
Comment (abompard) [reply devel/ci/integration/tests/test_bodhi_cli.py:482]: I think so, I'll remove them and give it a try.

File: devel/ci/integration/tests/test_bodhi_tasks.py (hunk 2 above)
Comment (abompard) [reply devel/ci/integration/tests/test_bodhi_tasks.py]: It's not, I'll remove the assignation, thanks.

**Comment by abompard** [comment]
As a very pedantic reviewer myself, I do appreciate the comments, thanks! I fixed these issues, let's see if CI passes now.

**Comment by abompard** [comment]
Alright, only F35 & Rawhide fail, as expected.

**Review by mattiaverga — APPROVED** [review]
LGTM thanks!

**Pull request merged**